public/app/features/dashboard/utils/getPanelMenu.ts

```diff
@@ -202,6 +206,77 @@ export function getPanelMenu(
     subMenu: inspectMenu,
   });
 
+  const createAlert = async () => {
+    const formValues = await panelToRuleFormValues(panel, dashboard);
+
+    const ruleFormUrl = urlUtil.renderUrl('/alerting/new', {
+      defaults: JSON.stringify(formValues),
+      returnTo: location.pathname + location.search,
+    });
+
+    locationService.push(ruleFormUrl);
+  };
+
+  const navigateToAlertListView = async () => {
+    const alertListUrl = urlUtil.renderUrl('/alerting/list', {
+      returnTo: location.pathname + location.search,
+    });
+
+    locationService.push(alertListUrl);
+  };
+
+  const onNavigateToAlertListView = (event: React.MouseEvent) => {
+    event.preventDefault();
+    navigateToAlertListView();
+    reportInteraction('dashboards_panelheader_menu', { item: 'create-alert' });
+  };
+
+  const onCreateAlert = (event: React.MouseEvent) => {
+    event.preventDefault();
+    createAlert();
+    reportInteraction('dashboards_panelheader_menu', { item: 'create-alert' });
+  };
+
+  const { unifiedAlertingEnabled } = getConfig();
+  const hasRuleReadPermissions = contextSrv.hasPermission(getRulesPermissions(GRAFANA_RULES_SOURCE_NAME).read);
+  const hasRuleUpdatePermissions = contextSrv.hasPermission(getRulesPermissions(GRAFANA_RULES_SOURCE_NAME).update);
+  const isAlertingAvailableForRead = unifiedAlertingEnabled && hasRuleReadPermissions;
+
+  const alertingMenuAvailable = isAlertingAvailableForRead;
+  if (alertingMenuAvailable) {
+    // prepare submenu depending on permissions
+    const subMenu: PanelMenuItem[] = [];
+    if (hasRuleUpdatePermissions) {
+      subMenu.push({
+        text: t('panel.header-menu.create-alert', `Create alert rule from this panel`),
+        onClick: (e: React.MouseEvent) => onCreateAlert(e),
+      });
+    }
+    subMenu.push({
+      text: t('panel.header-menu.view-alerts', `View all alert rules`),
+      onClick: (e: React.MouseEvent) => onNavigateToAlertListView(e),
+    });
+
+    menu.push({
+      type: 'submenu',
+      text: t('panel.header-menu.alerting', `Alerting`),
+      iconClassName: 'bell',
+      onClick: (e: React.MouseEvent<HTMLElement>) => {
+        const currentTarget = e.currentTarget;
+        const target = e.target;
```

Comment:
this menu is getting really busy, creating an alert rule from the dashboard view I think we do not need to support. From panel edit  sure but not from the "view dashboard" experiance

Reply:
I understand what you mean @torkelo ...The reason behind adding this menu, was that it's hard for users to fid out how to create an alert from a panel. Maybe we could find another way to help them creating alerts from here?

Reply:
> how to create an alert from a panel

Yea, I get that but we cannot add every possible action everywhere just to aid discoverability, as it hurts the experience of using/viewing the dashboard. 

You can place it under "...More" we do not want this menu to overwhelm the user with options, to make the core options of View, Edit, Share, and Inspect be clear and discoverable. 

I think we need to rename the "...More"  sub menu to be more of a "Send to" or "Create"  menu with actions that create things from a panel

Reply:
> we cannot add every possible action everywhere just to aid discoverability, as it hurts the experience of using/viewing the dashboard.
💯

> You can place it under "...More" 
sounds good!

Reply:
I've just moved the Alerting subMenu under the `More...` @torkelo 👀

<img width="717" alt="Screenshot 2023-10-17 at 17 22 16" src="https://github.com/grafana/grafana/assets/33540275/a3e320ab-487a-4cfe-9cdc-c9aa870f149c">

Reply:
I moved the Create alert and removed the View alert list @torkelo 👀
<img width="563" alt="Screenshot 2023-10-17 at 17 34 02" src="https://github.com/grafana/grafana/assets/33540275/dd53477c-30c4-48ef-8404-5589e495315d">